addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -193,6 +197,35 @@ public void initialize() {
         startAutomaticRefresh();
     }
 
+    @Override
+    public void handleConfigurationUpdate(Map<String, Object> configurationParameters) {
+        logger.debug("MAX! Cube {}: Configuration update received", getThing().getThingTypeUID());
+
+        Configuration configuration = editConfiguration();
+        for (Entry<String, Object> configurationParameter : configurationParameters.entrySet()) {
+            logger.debug("MAX! Cube {}: Configuration update {} to {}", getThing().getThingTypeUID(),
+                    configurationParameter.getKey(), configurationParameter.getValue());
+            if (configurationParameter.getKey().equals("cubeReboot")) {
+                if (configurationParameter.getValue().toString().equals("0")) {
+                    configurationParameter.setValue(BigDecimal.valueOf(-1));
+                    logger.info("Rebooting MAX! Cube {}", getThing().getThingTypeUID());
+                    MaxCubeBridgeConfiguration maxConfiguration = getConfigAs(MaxCubeBridgeConfiguration.class);
+                    UdpCubeCommand reset = new UdpCubeCommand(UdpCubeCommand.udpCommandType.RESET,
+                            maxConfiguration.serialNumber);
+                    reset.setIpAddress(maxConfiguration.ipAddress);
+                    reset.send();
+                }
+            }
+
+            configuration.put(configurationParameter.getKey(), configurationParameter.getValue());
+        }
+
+        // Persist changes
```

Comment:
The BaseThingHandler does it this way:
```
            // persist new configuration and notify Thing Manager
            updateConfiguration(configuration);
            callback.configurationUpdated(getThing());
```
The handleConfigurationUpdate is specifically introduced to NOT dispose/initialize the complete handler - so if you can avoid it, you should do so!